content/03.methods.md

```diff
@@ -377,16 +377,24 @@ High-grade glioma (HGG) subtypes were derived using the criteria below (addition
 5. If a sample was initially classified as HGAT, had no defining histone mutations, and a BRAF V600E mutation, it was subtyped as `BRAF V600E`.
 6. All other high-grade glioma samples that did not meet any of these criteria were subtyped as `HGG, H3 wildtype`.
 
-Non-MB and non-ATRT embryonal (`Embryonal tumor` in the `broad_histology` column of the metadata pbta-histologies.tsv) subtypes were derived using the criteria below [@pmid:30249036; @doi:10.1007/s00381-017-3551-6; @url:https://www.cancer.gov/types/brain/hp/child-cns-embryonal-treatment-pdq; @doi:10.3390/ijms21051818].
+Non-MB and non-ATRT embryonal tumors were subset from all embryonal tumor (`Embryonal tumor` in the `broad_histology` column of the metadata pbta-histologies.tsv) using the following criteria:
+1. An RNA-seq biospecimen sample includes a _TTYH1_ fusion (5' partner).
+2. An RNA-seq biospecimen sample includes a _MN1_ fusion (5' partner).
+Note that the `MN1--PATZ1` fusion is excluded as it is an entity separate of CNS HGNET-MN1 tumors [@doi:10.1111/nan.12626].
+3. Any sample with "Supratentorial or Spinal Cord PNET" or "Embryonal Tumor with Multilayered Rosettes" in the `pathology_diagnosis` column of the metadata `pbta-histologies.tsv`.
+4. Any sample with "Neuroblastoma" in the `pathology_diagnosis` column, where `primary_site` does not contain "Other locations NOS", `pathology_free_text_diagnosis` does not contain "peripheral" or "metastatic".
```

Comment:
Does "does not contain 'Other locations NOS'" mean is in the CNS? Because if so, we can just say that I think

```suggestion
4. A pathology diagnosis of "Neuroblastoma", where the tumor was not indicated to be peripheral or metastatic and was located in the CNS.
```

Reply:
Yes I believe it is so - I changed accordingly!